app/services/ipam/base.rb

```diff
@@ -0,0 +1,23 @@
+module IPAM
+  class Base
+    delegate :logger, :to => :Rails
+    attr_accessor :mac, :subnet, :excluded_ips
+
+    def initialize(opts = {})
+      @subnet = opts[:subnet]
+      @excluded_ips = opts.fetch(:excluded_ips, [])
+      @mac = opts.fetch(:mac, nil)
+
+      normalize_mac!
+    end
+
+    private
+
+    def normalize_mac!
+      return unless self.mac.present?
+      self.mac = Net::Validations.normalize_mac(self.mac)
+    rescue ArgumentError
```

Comment:
I realize you're just dealing with a bad exception name from `Net::Validations`, but `ArgumentError` is supposed to be used when there [are not enough arguments, too many, etc..](http://ruby-doc.org/core-2.3.1/ArgumentError.html), not for exceptions having to do with the code. 

There's another instance of this on `Host::Base.get_interface_scope`  and `Nic::Base.normalize_mac`. Can you change `normalize_mac` to raise `Error` and these methods to rescue `Net::Validations::Error` instead? It'll only get more confusing if we let this bad exception name creep in.